### PR TITLE
first att

### PR DESCRIPTION
**Pull request by CarlosRabid**
first atempt

### CODE REVIEW

**Comment by neb-sidnal** [comment]
Barbara Streisand

**Comment by Luis270100** [comment]
Nice

**Review by pagella42 — COMMENTED** [review]
will only approve if i get payed

File: dist/main.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+let x = 3
```
Comment (Axel936) [inline dist/main.js:1]: Look, Boobs
!! >>>>>> (.)(.)

**Pull request merged**